### PR TITLE
FlxG: Add FlxG.onMobile

### PR DESCRIPTION
**Pull request by DleanJeans**
Also I renamed `isMobile` in HTML5FrontEnd to `onMobile` since it sounds better.

### CODE REVIEW

File: flixel/FlxG.hx [hunk 1]
```diff
@@ -707,6 +709,22 @@ class FlxG
 	{
 		return game._state;
 	}
+	
+	private static inline function get_onMobile():Bool
+	{
+		if (onMobile == null)
```
Comment (MSGhero) [inline flixel/FlxG.hx]: What's the point of this check? Why not make it `@:isVar public static var onMobile(get, never):Bool` and
```haxe
private static inline function get_onMobile():Bool {
   return #if js blah #elseif mobile true #else false #end;
}
```

**Comment by DleanJeans** [comment]
Not sure why thus

**Comment by DleanJeans** [comment]
Not sure why Travis didn't build. It's been working OK in my project so fast.

**Comment by MSGhero** [comment]
Travis fails every project for the same reason, something weird on cpp and neko.

**Comment by DleanJeans** [comment]
All the builds were successful but still exited with 1?

**Comment by MSGhero** [comment]
Yeah, if you look closely,
```
X Error of failed request:  BadRequest (invalid request code or no such operation)
  Major opcode of failed request:  147 (GLX)
  Minor opcode of failed request:  168 ()
  Serial number of failed request:  173
  Current serial number in output stream:  173
```
on Haxe 3.2.0 neko and Haxe dev neko/cpp for every recent pull request. This pull is fine since all the demos and coverages passed.

**Comment by Gama11** [comment]
Thanks!

I'm not a huge fan of putting even more stuff into `FlxG`, but I don't really have a better alternative right now and this does seem useful.

The Travis failures are indeed unrelated, I'm noto sure what's causing them yet.

I don't like the formatting of `get_onMobile()`'s body, cramping everything into one line doesn't make it very readable.

**Pull request merged**